### PR TITLE
Addressing prerelease version comments

### PR DESCRIPTION
**Pull request by tpatpa**
Schema attributes were added to better describe the behavior of the schema elements and improve clinical usability.

The schema was validated and converted using HED online tools.

### CODE REVIEW

File: library_schemas/score/prerelease/HED_score_1.0.0.mediawiki [hunk 1]
```diff
@@ -56,35 +56,35 @@ This project was supported by the by the National Institute of Mental Health of
         ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
     * Cognitive-task
         ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Other-modulator-or-procedure
+    * Other-modulator-or-procedure {requireChild,extensionAllowed}
```
Comment (VisLab) [inline library_schemas/score/prerelease/HED_score_1.0.0.mediawiki:59]: You should take out the extensionAllowed.  The # and extensionAllowed are conflicts. You cannot add another node below Other-modulator-or-procedure.  The # says that you can add a value.

Comment (tpatpa) [reply library_schemas/score/prerelease/HED_score_1.0.0.mediawiki:59]: Thanks for pointing that out!
So either a childe node that takes value like this:

> <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>

or the extensionAllowed schema attribute?

Comment (dorahermes) [reply library_schemas/score/prerelease/HED_score_1.0.0.mediawiki:59]: This is still a **draft** pull request correct? Let me know when you want me to comment.

**Comment by tpatpa** [comment]
In the [SCORE 2nd version](https://doi.org/10.1016/j.clinph.2017.07.418) Posterior Dominant Rhythm (PDR) is described as the most often annotated EEG feature in clinical practice. Therefore SCORE terminology includes additional terms to this feature.

All PDR terms in the schema were reviewed, and changes were made in [this commit](https://github.com/hed-standard/hed-schema-library/pull/27/commits/17e1070938bcb5fa69d49ec3e3bdbc07cb286524). The following terms were not changed as they are unique to PDR:
 - rhythm-organization
 - amplitude-range is 
 - eye-opening-reactivity
 - caveat
 - Absence-of-posterior-dominant-rhythm

**Comment by VisLab** [comment]
This PR has been discussed extensively.  We need to merge as a prerelease so that it can be viewed in the schema viewer and finalized.

**Pull request merged**